plugins/infinispan52/src/main/java/org/radargun/service/InfinispanServerService.java

```diff
@@ -159,7 +159,7 @@ public InfinispanServerClustered getClustered() {
    @Override
    protected List<String> getCommand() {
       ArrayList<String> command = new ArrayList<String>();
-      command.add(FileSystems.getDefault().getPath(home, "bin", "standalone." + (windows ? "bat" : "sh")).toString());
+      command.add(FileSystems.getDefault().getPath(home, "bin", "clustered." + (windows ? "bat" : "sh")).toString());
```

Comment:
Sure @rvansa, I can see that the standalone and clustered scripts are basically identical, but I think it is important that the servers use the default clustered configuration if the configuration file is not specified in the benchmark.